src/app/core/entity/entity-mapper.service.ts

```diff
@@ -105,6 +136,24 @@ export class EntityMapperService {
    * @param entity The entity to be deleted
    */
   public remove<T extends Entity>(entity: T): Promise<any> {
+    this.sendUpdate(entity, "remove");
```

Comment:
I would consider using a enum for `remove`, `update` and `new`

Reply:
The current type is a literal union type. Is there an advantage exchanging this for an enum?